org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/StatusBar.java

```diff
@@ -173,11 +175,15 @@ public void setStatusLine (String text, boolean error, ProcessInfoLog[] m_logs)
 			return;
 		
     	String labelText = buildLabelText(m_statusText);
+    	//iDempiereConsulting __05/10/2021 --- Error Message - The popup exposes the complete message and life time can be configured through system variable configurator (ErrorMsgLifeTime)  
+    	// If the value equals '0', then the popup will not disappear and you will need to close it manually; otherwise set the variable as milleseconds
+    	int duration = Integer.parseInt(MSysConfig.getValue("ErrorMsgLifeTime", "4500", Env.getAD_Client_ID(Env.getCtx())));   //MSysConfig.getIntValue("ErrorMsgLifeTime", 3500);
     	if (error) {
-    		Notification.show(buildNotificationText(m_statusText), "error", findTabpanel(this), "top_left", 3500, true);
+    		Notification.show(m_statusText, "error", findTabpanel(this), "top_left", duration, true);
```

Comment:
why replace "buildNotificationText(m_statusText)" with "m_statusText" ? this is not explain in the ticket.